frontend/javascripts/admin/organization/upgrade_plan_modal.tsx

```diff
@@ -341,7 +343,7 @@ function OrderWebknossosCreditsModal({ destroy }: { destroy: () => void }) {
       <div className="drawing-upgrade-users">
         <p style={{ marginRight: "30%" }}>
           You can buy new WEBKNOSSOS credits to pay for premium jobs and services. Each credit costs
-          5€ or 5.75$.
+          {formatCurrency(5, "€")} or {formatCurrency(5.75, "$")}.
```

Comment:
also needs extraction probably

Reply:
I am a little unsure what you meant here with extraction. Using the credits from the features?